plugins/newsdownloader.koplugin/feed_config.lua

```diff
@@ -0,0 +1,8 @@
+return {
+    -- list your feeds here:
+    -- only supports http URL for now
+    -- Atom is currently not supported, only RSS
+    -- set limit to "0" means no download, "-1" no limit.
+    { "http://www.huffingtonpost.com/feeds/index.xml", limit = 2 },
```

Comment:
AP relays on injection XMLs into pages, so it wasn't correct on our simple "wget" we do for download.

I changed that to economist science and culture(disabled by default) as they should be neutral.
Fell free to update it to something better that's international, RSS & neutral.